utils/build/docker/ruby/parametric/server.rb

```diff
@@ -29,7 +29,7 @@
 OpenTelemetry::SDK.configure # Initialize OpenTelemetry
 
 Datadog.configure do |c|
-  c.diagnostics.debug = true # When tests fail, ensure there's enough data to debug the failure.
+  # c.diagnostics.debug = true # When tests fail, ensure there's enough data to debug the failure.
```

Comment:
Enabling debug logging here interferes with the test. We have to find a workaround for this. Maybe update the test cases?